pkg/controller/keycloakrealm/keycloakrealm_reconciler_test.go

```diff
@@ -57,6 +57,53 @@ func getDummyRealm() *v1alpha1.KeycloakRealm {
 	}
 }
 
+func getUpdatedRealm() *v1alpha1.KeycloakRealm {
+	return &v1alpha1.KeycloakRealm{
+		Spec: v1alpha1.KeycloakRealmSpec{
+			InstanceSelector: &v1.LabelSelector{
+				MatchLabels: map[string]string{
+					"app": "keycloak",
+				},
+			},
+			RealmOverrides: []*v1alpha1.RedirectorIdentityProviderOverride{
+				{
+					IdentityProvider: "openshift-v4",
+					ForFlow:          "browser",
+				},
+			},
+			Realm: &v1alpha1.KeycloakAPIRealm{
+				ID:                        "dummy",
+				Realm:                     "dummy",
+				Enabled:                   true,
+				DisplayName:               "updated",
+				EventsEnabled:             &[]bool{false}[0],
+				AdminEventsEnabled:        &[]bool{false}[0],
+				AdminEventsDetailsEnabled: &[]bool{false}[0],
+				Attributes: map[string]string{
```

Comment:
Related to your previous scenario. Does this test cover all updates cases? (addition of value, change in value, deletion of value) and check for all those cases?

Reply:
It currently tests updating values and adding an attrribute